lib/katello/permissions/ui_permissions.rb

```diff
@@ -0,0 +1,5 @@
+require 'katello/plugin'
+
+Foreman::Plugin.find(:katello).security_block :katello_ui do
+  permission :view_katello_ui, { 'bastion/bastion' => [:index, :index_ie] }
```

Comment:
The routes added by bastion. I am really not sure if they should have a permission as well or be excluded from from permission check on foreman side.